### DISCUSSION TITLE
Only 14 tweets output

### DISCUSSION THREAD

**Discussion by IParraMartin** [tweepy/tweepy#1573] [Questions] [answered]
I am currently working on a Sentiment Analysis project for my degree thesis. I am using tweepy to access a range of tweets that allow me to carry my research but the output keeps being 14 tweets instead of 99-100. 

I am using **public tweets = api.search** instead of **user_timeline** to have a general database of Covid related tweets. 
Why can’t I get more tweets in the output?

Please, I need help ASAP.

**Comment by Harmon758** [comment, marked as answer]
You'll want to use the `count` parameter of [`API.search`](https://tweepy.readthedocs.io/en/latest/api.html#tweepy.API.search):
> The number of tweets to return per page, up to a maximum of 100. Defaults to 15.

https://developer.twitter.com/en/docs/twitter-api/v1/tweets/search/api-reference/get-search-tweets

**Reply by IParraMartin** [reply]
I am a beginner at this, how can I implement that parameter? Just adding **'count = 100'**?

**Reply by Harmon758** [reply]
You'll want to pass it as a keyword argument, yes.
https://docs.python.org/3/tutorial/controlflow.html#keyword-arguments

**Reply by IParraMartin** [reply]
I never had the time to reply to the help people offered me when I was still a real beginner at coding and NLP. Thanks so much for helping me when nobody was willing to! This is what drives people to keep learning! 🚀🥲😊
